package.json

```diff
@@ -175,11 +177,11 @@
   "bundlesize": [
     {
       "path": "lib/**/*.js",
-      "maxSize": "45 kB"
+      "maxSize": "46 kB"
     },
     {
       "path": "commerce-sdk-isomorphic-with-deps.tgz",
-      "maxSize": "350 kB"
+      "maxSize": "400 kB"
```

Comment:
Is this because that lib has a lot of docs or large source that is being added to the gzip?

Reply:
I assume that's the reason as you can see in the previous [Joel's comment](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/150#pullrequestreview-2034095413) comparing the before and after file sizes. We added two libraries `seedrandom` as a dependency and `@types/seedrandom` as a devDependency.

But as later [Joel also mentions](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/150#pullrequestreview-2036226789) the minimized bundle size is the size we care about and that one only increased slightly, 1KB.